pubpub/platform
# Render URLs as links

## Issue(s) Resolved
Closes https://github.com/pubpub/platform/issues/843

## High-level Explanation of PR
When rendering pub values, now we check that the `schemaName` is of `CoreSchemaType.URL`, and if so we render `<a>` tags

## Test Plan
* Add a pub with a pub value of type URL (or look at the seeded croc croc pub which has a URL field)
* Make sure that the value renders as a link and when you click the link it opens in a new tab

## Screenshots (if applicable)

## Notes
